packages/rocketchat-ui-message/message/renderMessageBody.js

```diff
@@ -21,6 +21,36 @@ renderMessageBody = function(msg) {
 	}
 
 	msg.html = message.html.replace(/\n/gm, '<br/>');
+	
+	if(singleEmojiCheck(msg))
+		msg.html = (msg.html).replace("emojione ", "big_emojione ");
 
 	return msg.html;
 };
+
+/**
+ *Checks to see if a message contains only an emoji span
+ *@para {msg} some html message
+ *@return {false} the html message either doesn't contain an emoji span, or contains other characters
+ */
+singleEmojiCheck = function(msg){
+	var str = msg.html;
+	str = str.trim();
+	//checks to make sure it's a span
+	if(!str.includes("<") || !str.includes(">"))
+		return false;
+	if(str.charAt(0) != '<' || str.charAt(str.length - 1) != '>')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2486979418/issues/source?bid=3199634&fileBranchId=3266617#l42)